File: 프로그래머스/lv1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    answer = True
    
    # [실행] 버튼을 누르면 출력 값을 볼 수 있습니다.
    print('Hello Python')
    p_num,y_num=0,0
    new=s.upper()
    for i in range(len(s)):
        if new[i] == 'P':
            p_num+=1
        elif new[i] == 'Y':
            y_num+=1
    
    if p_num == y_num:
        return True
    elif p_num + y_num == 0:
        return True
    else:
        return False

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.js
function solution(s){
    var answer = true;
    
    let count_p = 0
    let count_y = 0
    
    s = s.toLowerCase()
    for(let i=0; i<s.length;i++){
        if(s[i] === 'p'){
            count_p+=1
        } else if (s[i] === 'y') {
            count_y+=1
        }
    }
    
    if(count_p !== count_y){
        return false
    }
    return answer;
}